src/main/scala/com/evvo/remoting/RemoteActorSystem.scala

```diff
@@ -8,9 +8,8 @@ import com.typesafe.config.ConfigFactory
 object RemoteActorSystem {
   def main(args: Array[String]): Unit = {
 
-    val configFile = ConfigFactory.parseFile(new File("src/main/resources/application.conf"))
-    val config = configFile
-      .withFallback(configFile)
```

Comment:
parsing config with fallback to the same file doesn't make sense, so I removed it, just use application.conf